babybuddy/templates/babybuddy/base.html

```diff
@@ -47,6 +47,23 @@
         <script>BabyBuddy.PullToRefresh.init()</script>
     {% endif %}
 
-{% block javascript %}{% endblock %}
+{% block javascript %}
+
+<script type="application/javascript">
+    function rememberAdvancedToggle(event, element){
+        localStorage.setItem("advancedForm",event.newState)
+    }
+
+    window.addEventListener('load', function () {
+        if(localStorage.getItem("advancedForm") !== "open"){
+            return;
+        }
+
+        document.querySelectorAll("form details").forEach(function(node){
+            node.open = true
+        });
+    })
+</script>
```

Comment:
This does feel a bit awkward but I think it works well enough for a simple use case. Lets move this code in to the babybuddy,js file and namespace though.